notebooks/CartoPy/CartoPy.ipynb

```diff
@@ -375,10 +375,11 @@
     "# Create some synthetic gridded wind data\n",
     "import numpy as np\n",
     "from metpy.calc import wind_speed\n",
+    "from metpy.units import units\n",
     "\n",
     "# Note that all of these winds have u = 0 -> south wind\n",
-    "v = np.full((5, 5), 10, dtype=np.float64) + 10 * np.arange(5)\n",
-    "u = np.zeros_like(v)\n",
+    "v = np.full((5, 5), 10, dtype=np.float64) + 10 * np.arange(5) * units('knots')\n",
```

Comment:
I'm just curious: why `units('knots')` and not `units.knots`? I usually reserve the parser ones for when it's a compound unit that would otherwise be verbose to write.

Reply:
Didn't know there was a preference here. Easy to change - just what I had in my mind at the time but no particular reason.